packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -205,6 +226,10 @@ export class DataVisualizerGraph
 		return this.visualizerNodes.get(fluidObjectId)?.render() ?? undefined;
 	}
 
+	public async preformEdit(fluidObjectId: FluidObjectId, newData: Serializable): Promise<void> {
```

Comment:
Nitpick: What do you think of "applyEdit"?

Reply:
Agree. My brain was reading this as `performEdit` for a while 😄 .

Reply:
Check new commit